marklogic-data-hub/src/test/java/com/marklogic/hub/impl/ClearUserArtifactsTest.java

```diff
@@ -0,0 +1,40 @@
+package com.marklogic.hub.impl;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.marklogic.hub.AbstractHubCoreTest;
+import com.marklogic.hub.test.ReferenceModelProject;
+import org.junit.jupiter.api.Assumptions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ClearUserArtifactsTest extends AbstractHubCoreTest {
+    int initialDbCount;
+    @BeforeEach
+    void setUp() {
+        Assumptions.assumeTrue(isVersionCompatibleWith520Roles());
+        initialDbCount = Integer.parseInt(getHubClient().getStagingClient().newServerEval().javascript("cts.estimate(cts.trueQuery())").evalAs(String.class));
+        // These 2 projects contain all kinds of user artifacts
+        installProjectInFolder("test-projects/download-artifacts");
+        ReferenceModelProject project = installReferenceModelProject();
+        project.createRawCustomer(1, "Jane");
+    }
+
+    @Test
+    void clearUserArtifacts() {
+        try{
+            new DataHubImpl(runAsDataHubOperator()).clearUserArtifacts();
+            fail("'data-hub-operator' should not be able delete user artifacts");
+        }
+        catch (Exception e){
+            logger.info("'data-hub-operator' cannot delete user artifacts");
```

Comment:
A good practice here is to make an assertion on what the exception message contains - that way, we know we're catching the expected exception. Can also catch a more precise exception here - I assume that ML is throwing a FailedRequestException.